tests/syntax/test_compiler.py

```diff
@@ -1,4 +1,6 @@
 from ast import *
+
+import pytest
 from scenic.syntax.ast import *
```

Comment:
Let's move this down one line. I like to keep the imports separated into 3 blocks in general: standard Python libraries, other dependencies, then Scenic modules.

Reply:
199cc9e52bf76f5308b0b7ebeec1b26cdcf2df76

Fixed the same issues for newly added files